INSTALL.md

```diff
@@ -0,0 +1,121 @@
+# Ceph Telemetry Installation
+
+## Minimum requirements
+- RHEL 8 based OS

```

Comment:
how hard a requirement is this?  Will, say, CentOS Stream work?

Reply:
currently mod_evasive is not available in CentOS Stream, which is a problem